.github/workflows/build_pr_documentation.yml

```diff
@@ -15,3 +15,5 @@ jobs:
       pr_number: ${{ github.event.number }}
       package: transformers
       languages: de en es fr it ko pt zh
+    secrets:
+      hf_token: ${{ secrets.HF_DOC_BUILD_PUSH }}
```

Comment:
PRs from forks cannot use secrets from the repo, whatever you do.

Reply:
With `pull_request_target` maybe?